ManagementPacks/SquaredUp.EAM.Samples/SampleEAs/AvailabilityOnly/Discovery.mpx

```diff
@@ -0,0 +1,205 @@
+          <MembershipRules>
+            <MembershipRule>
+              <MonitoringClass>$MPElement[Name="EAM!SquaredUp.EAM.Library.Class.AvailabilityMonitoring.Watcher"]$</MonitoringClass>
+              <RelationshipClass>$MPElement[Name="MSIL!Microsoft.SystemCenter.InstanceGroupContainsEntities"]$</RelationshipClass>
+              <Expression>
+                <SimpleExpression>
+                  <ValueExpression>
+                    <Property>$MPElement[Name="System!System.Entity"]/DisplayName$</Property>
+                  </ValueExpression>
+                  <Operator>Equal</Operator>
+                  <ValueExpression>
+                    <Value>Sample - Tailspin Toys</Value>
+                  </ValueExpression>
+                </SimpleExpression>
+              </Expression>
+            </MembershipRule>
+          </MembershipRules>
+        </DataSource>
+      </Discovery>
+      <Discovery ID="SquaredUp.EAM.Samples.Class.AvailabilityOnly.Watcher.Discovery" Enabled="true" Target="EAM!SquaredUp.EAM.Library.Class.AvailabilityMonitoring.MonitoringNode" ConfirmDelivery="false" Remotable="true" Priority="Normal">
+        <Category>Discovery</Category>
+        <DiscoveryTypes>
+          <DiscoveryClass TypeID="EAM!SquaredUp.EAM.Library.Class.AvailabilityMonitoring.Watcher.Web">
+            <Property TypeID="EAM!SquaredUp.EAM.Library.Class.AvailabilityMonitoring.Watcher" PropertyID="ConfigurationJson" />
+            <Property TypeID="EAM!SquaredUp.EAM.Library.Class.AvailabilityMonitoring.Watcher" PropertyID="ApplicationName" />
+            <Property TypeID="EAM!SquaredUp.EAM.Library.Class.AvailabilityMonitoring.Watcher" PropertyID="ResponseTimeThreshold" />
+          </DiscoveryClass>
+        </DiscoveryTypes>
+        <DataSource ID="DS" TypeID="EAM!SquaredUp.EAM.Library.DataSource.AvailabilityMonitoring.StaticAlwaysOnDiscovery">
+          <IntervalSeconds>86400</IntervalSeconds>
+          <DiscoverInstance>false</DiscoverInstance>
+          <ComputerName>$Target/Host/Property[Type='Windows!Microsoft.Windows.Computer']/PrincipalName$</ComputerName>
+          <ClassId>$MPElement[Name='EAM!SquaredUp.EAM.Library.Class.AvailabilityMonitoring.Watcher.Web']$</ClassId>
+          <InstanceSettings>
+            <Settings>
+              <Setting>
+                <Name>$MPElement[Name="Windows!Microsoft.Windows.Computer"]/PrincipalName$</Name>
+                <Value>$Target/Host/Property[Type="Windows!Microsoft.Windows.Computer"]/PrincipalName$</Value>
+              </Setting>
+              <Setting>
+                <Name>$MPElement[Name='System!System.Entity']/DisplayName$</Name>
+                <Value>Sample - Tailspin Toys</Value>
+              </Setting>
+              <Setting>
+                <Name>$MPElement[Name='EAM!SquaredUp.EAM.Library.Class.AvailabilityMonitoring.Watcher']/ConfigurationJson$</Name>
+                <Value>{"url":"https://tailspintoys.com","match":"Tailspin","notmatch":"404"}</Value>
```

Comment:
...so "https://tailspintoys.com" is one of MS's fictional outfits, right? It's interesting to see that there are now several real-looking toy resellers trading under that name!